python/doc/bibliography.rst

```diff
@@ -14,6 +14,9 @@ Bibliography
     *Lilliefors/Van Soest’s test of normality.*. Encyclopedia of Measurement and Statistics, 2007.
 .. [AbdiMolin1998] Hervé Abdi, Paul Molin.
     *New table and numerical approximations for approximations for Kolmogorov-Smirnov / Lillifors / Van Soest normality test.*, 1998.
+.. [acklam2017] Acklam P.J.
+    *Acklam's algorithm for the inverse normal cdf*, 2017.
+     https://stackedboxes.org/2017/05/01/acklams-normal-quantile-function/
```

Comment:
looks it errors on the 1 extra space indent

Reply:
yes... thanks